.buildkite/pipeline.yml

```diff
@@ -0,0 +1,15 @@
+steps:
+  - label: "Publish share_snapshots_rds lambda"
+    # Note, Buildkite passes the script around as an ENV VAR so any local variables
+    # need to be escaped
+    command: |
+      cd lambda
+      for component in take share delete_old; do
```

Comment:
We need to publish each component since we're updating the bucket and the existing CFN assumes they're all in the same bucket